src/status_im/ui/screens/chat/input/input.cljs

```diff
@@ -89,7 +89,6 @@
        :auto-capitalize        :sentences}
       (when cooldown-enabled?
         {:placeholder (i18n/label :cooldown/text-input-disabled)}))]))
-;)
```

Comment:
?

Reply:
I removed commented out brace because it is leftover and it shouldn't be there.